src/executePrettier/buildPrettierOptions.test.js

```diff
@@ -109,40 +107,46 @@ it('uses vue as the parser if current scope is listed as a Vue SFC scope in sett
   expect(actual).toEqual({ parser: 'vue' });
 });
 
-it('does not use editorconfig options if that setting is not enabled', () => {
+it('passes editorconfig option to prettier', () => {
+  let editorconfig = false;
+
   const editor = buildMockEditor();
+  const mockPrettierInstance = { resolveConfig: { sync: jest.fn() } };
+  getPrettierInstance.mockImplementation(() => mockPrettierInstance);
+  shouldUseEditorConfig.mockImplementation(() => editorconfig);
+  getCurrentFilePath.mockImplementation(() => 'parent-dir/foo.js');
   getPrettierOptions.mockImplementation(() => ({}));
-  shouldUseEditorConfig.mockImplementation(() => false);
-
   buildPrettierOptions(editor);
+  expect(mockPrettierInstance.resolveConfig.sync.mock.calls[0][1]).toEqual({ editorconfig });
 
-  expect(buildEditorConfigOptions).not.toHaveBeenCalled();
+  editorconfig = true;
+  buildPrettierOptions(editor);
+  expect(mockPrettierInstance.resolveConfig.sync.mock.calls[1][1]).toEqual({ editorconfig });
 });
 
-it('overrides values with editorconfig values if editor config is enabled', () => {
+it('only uses project prettier config when found', () => {
```

Comment:
This makes sense in the context of this PR, but in the future, some may not understand what we mean by" only uses project prettier config." I think we should change to `uses settings from the prettier config if found`, and then we can make the test below use `uses settings from prettier-atom if the prettier config is not found`.

Reply:
good point